ivy/data_classes/array/experimental/norms.py

```diff
@@ -39,93 +39,123 @@ def l2_normalize(
         """
         return ivy.l2_normalize(self, axis=axis, out=out)
 
+    def batch_norm(
+        self,
+        mean: Union[ivy.NativeArray, ivy.Array],
+        variance: Union[ivy.NativeArray, ivy.Array],
+        /,
+        *,
+        offset: Optional[Union[ivy.NativeArray, ivy.Array]] = None,
+        scale: Optional[Union[ivy.NativeArray, ivy.Array]] = None,
+        training: bool = False,
+        eps: float = 1e-5,
+        momentum: float = 1e-1,
+        out: Optional[ivy.Array] = None,
+    ) -> ivy.Array:
+        """
+        ivy.Array instance method variant of ivy.batch_norm. This method
+        simply wraps the function, and so the docstring for ivy.batch_norm
+        also applies to this method with minimal changes.
+
+        Parameters
+        ----------
+        self
+            Input array of shape (N,C,S), where N is the batch dimension, C is the
+            feature dimension and S corresponds to the following spatial dimensions.
+        mean
+            A mean array for the input's normalization.
+        variance
+            A variance array for the input's normalization.
+        offset
+            An offset array. If present, will be added to the normalized input.
+        scale
+            A scale array. If present, the scale is applied to the normalized input.
+        training
+            If true, calculate and use the mean and variance of `x`. Otherwise, use the
+            provided `mean` and `variance`.
+        eps
+            A small float number to avoid dividing by 0.
+        momentum
+             The value used for the running_mean and running_var computation.
```

Comment:
I don't think you've mentioned running_mean and running_var anywhere previously. Would be nice to rephrase the description a bit to explain what these are. The same goes for all the added functions.

Reply:
Updated :+1: